lib/generators/draft/resource/templates/controllers/controller.rb

```diff
@@ -73,86 +43,50 @@ def create
-        end
-  
-        format.html do
-          redirect_to("/<%= @plural_table_name.underscore %>/#{@<%= singular_table_name.underscore %>.id}", {:notice => "<%= singular_table_name.humanize %> updated successfully."})
-        end
-      end
+      redirect_to("/<%= @plural_table_name.underscore %>/#{@<%= singular_table_name.underscore %>.id}", {:notice => "<%= singular_table_name.humanize %> updated successfully."})
     else
-      # render({:template => "/<%= @plural_table_name.underscore %>/edit_form_with_errors.html.erb"})
-      respond_to do |format|
-        format.json do
-          render({ :json => @<%= singular_table_name.underscore %>.as_json })
-        end
-  
-        format.html do
-          render({ :template => "<%= plural_table_name.underscore %>/show.html.erb" })
-        end
-      end
+      render({ :template => "<%= plural_table_name.underscore %>/show.html.erb" })
     end
 <% else -%>
     @<%= singular_table_name.underscore %>.save
 
 <% unless skip_redirect? -%>
     redirect_to("/<%= @plural_table_name.underscore %>/#{@<%= singular_table_name.underscore %>.id}")
 <% else -%>
-  respond_to do |format|
-    format.json do
-      render({ :json => @<%= singular_table_name.underscore %>.as_json })
-    end
-
-    format.html do
       render({ :template => "<%= plural_table_name.underscore %>/show.html.erb" })
-    end<% end -%>
+  <% end -%>
 <% end -%>
   end
 
   def destroy
-    the_id = params.fetch(:rt_<%= singular_table_name %>_id)
-    @<%= singular_table_name.underscore %> = <%= class_name.singularize %>.where({ :id => the_id }).first
+    the_id = params.fetch("id_from_path")
+    @<%= singular_table_name.underscore %> = <%= class_name.singularize %>.where({ :id => the_id }).at(0)
 
     @<%= singular_table_name.underscore %>.destroy
 
 <% unless skip_validation_alerts? -%>
-  respond_to do |format|
-    format.json do
-      render({ :json => @<%= singular_table_name.underscore %>.as_json })
-    end
-
-    format.html do
-      redirect_to("/<%= @plural_table_name.underscore %>", {:notice => "<%= singular_table_name.humanize %> deleted successfully."})
-    end
-  end
+    redirect_to("/<%= @plural_table_name.underscore %>", {:notice => "<%= singular_table_name.humanize %> deleted successfully."})
 <% else -%>
 <% unless skip_redirect? -%>
     redirect_to("/<%= @plural_table_name.underscore %>")
 <% else -%>
     @remaining_count = <%= class_name.singularize %>.count
     
-    respond_to do |format|
-      format.json do
-        render({ :json => @<%= singular_table_name.underscore %>.as_json })
-      end
-
-      format.html do
-        redirect_to("/<%= @plural_table_name.underscore %>", {:notice => "<%= singular_table_name.humanize %> deleted successfully."})
-      end
-    end
-    render("<%= singular_table_name.underscore %>_templates/destroy_row.html.erb")
+    redirect_to("/<%= @plural_table_name.underscore %>", {:notice => "<%= singular_table_name.humanize %> deleted successfully."})    
```

Comment:
I think we need space 
```suggestion
    redirect_to("/<%= @plural_table_name.underscore %>", { :notice => "<%= singular_table_name.humanize %> deleted successfully." })    
```